src/components/Layout/PageTransition.tsx

```diff
@@ -1,12 +1,20 @@
 import { useMediaQuery } from '@chakra-ui/media-query'
 import { HTMLMotionProps, motion } from 'framer-motion'
+import { useWallet } from 'hooks/useWallet/useWallet'
 import { breakpoints } from 'theme/theme'
 
 export const PageTransition = (props: HTMLMotionProps<'div'>) => {
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`)
+  const walletContext = useWallet()
+  const minHeight = isLargerThanMd
```

Comment:
@reallybeard are you ok with this?

Reply:
Hi @reallybeard, I know this options isn't ideal, but this was the only way I could find to maintain existing behavior exactly. 
I am certainly open to suggestions, particularly if you know of a way to pass down a reference to the child components so the height could be queried at runtime?
Alternatively, I did also come up with this `initial={{ y: -16, opacity: 0, minHeight: 'auto' }}`
While is does not preserve the "fullscreen" scroll height, this does not seem to be noticeable from a user's perspective. Since the background color matches that of the element and the scroll height is auto scaled, this does seem to result in the same observable outcome as the current implementation. Unless, of course, there is something here that I missed?

Reply:
Yah this is fine, we need to offset the height to account for the banner.